pandas/core/dtypes/concat.py

```diff
@@ -242,6 +242,72 @@ def union_categoricals(to_union, sort_categories=False, ignore_order=False):
         - sort_categories=True and Categoricals are ordered
     ValueError
         Empty list of categoricals passed
+
+    Notes
+    -----
+
+    To learn more about categories, see 'link
```

Comment:
see `here .....`

and the ``__`` (at the end) should be *outside* the single quote